CHANGELOG.md

```diff
@@ -4,6 +4,126 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project will adhere to [Calendar Versioning](https://calver.org/) starting v20.03.
 
+## [20.07.2] - Unreleased
+
+[20.07.2]: https://github.com/dgraph-io/dgraph/compare/v20.07.1...v20.03.2
+
+### Changed
+
+- Updater badger to 5e3d4b9 (#6669)
+- docs: Update restore documentation. (#6653)
+- doc: update /admin endpoint documentation. (#6415) (#6527)
+- test: Enterprise License tests (#5822) (#6640)
+- build: Makefile to build Dgraph inside docker container. (#6601)
+- docs: add documentation for clone backend (#6622)
+- chore(query): Return content length header for queries (#6469) (#6480)
+- perf(raft): Use raft storage in managedmode (#6457) (#6547)
+- Update index.md (#6566) (#6567)
+- Changes github.com/dgraph-io/graphql-transport-ws version (#6462) (#6529)
+- docs: Update list documentation (#6455) (#6461)
+
+### Added
+
+- feat(Dgraph): add utility to export backup data. (#6590)
+- feat(Dgraph): Add separate compression flag for z and wal dirs (#6401) (#6421)
+
+### Fixed
+
+- fix(Auth): fix Poor-man's auth for admin operations (#6660) (#6686)
+- Break out if g.Ctx is done (#6675)
+- fix(Query): Fix wrong path response for k-shortest paths (#6437) (#6654)
+- fix (graphql): disallowing field names with as (#6645)
+- fix(GraphQL Query): Remove extra fields when querying interfaces (#6596) (#6647)
```

Comment:
please put all GraphQL changes together so we can stay consistent from previous versions.